test/roblox-cli.sh

```diff
@@ -15,4 +15,4 @@ else
 fi
 
 rojo build test-place.project.json -o TestPlace.rbxlx
-roblox-cli run --load.place TestPlace.rbxlx --assetFolder "$CONTENT"
\ No newline at end of file
+robloxdev-cli run --load.place TestPlace.rbxlx --assetFolder "$CONTENT"
```

Comment:
Why set to robloxdev-cli? Since its a public repository we may want to try to keep this as roblox-cli if possible